src/Profile.php

```diff
@@ -1169,6 +1169,12 @@ public static function getRightsForForm(string $interface = 'all', string $form
                     ]
                 ]
             ];
+
+            // Add rights for custom assets
+            $definitions = \Glpi\Asset\AssetDefinitionManager::getInstance()->getDefinitions();
+            foreach ($definitions as $definition) {
```

Comment:
@orthagh Should we display inactive definitions here?

Reply:
I'm not sure. I'm inclined on no.